cloud/datasync/src/web/SyncWebExpress.ts

```diff
@@ -23,7 +22,7 @@ export class SyncExpressMiddleware {
    * Create express router for sync endpoints
    */
   public createSyncExpressRouter() {
-    const apiURI = path.join(this.prefix + '/:datasetId');
+    const apiURI = this.prefix + '/:datasetId';
```

Comment:
:+1: so it seems that the right method URLs would be [`url.resolve`](https://nodejs.org/api/url.html#url_url_resolve_from_to) to avoid problems in windows.

I've did a quick search through the codebase and I think this is the only instance where this method is being used with urls.

Reply:
I copied over example with url resolve, but it's not valid for this use case: 
```
url.resolve('/one/two/three', 'four');         // '/one/two/four'
```

Simple concatenation will do the job here.

Reply:
Yeah, considering that we were already passing the `+` concatenated value to `path.join` it wasn't even doing anything before except wrongly replacing the directory separator in windows.